share/sysbuild/cmake/modules/sysbuild_extensions.cmake

```diff
@@ -251,6 +251,9 @@ function(ExternalZephyrProject_Add)
       menuconfig
       hardenconfig
       guiconfig
+      ram_report
+      rom_report
+      footprint
```

Comment:
you are placing those inside a `foreach` where the loop variable is name `kconfig_target`.

If we should start exposing more targets from image build system, then the loop var should be renamed.

Reply:
Would it make sense to have these in a centralised file somewhere to list what possible targets are supported (and dependent Kconfigs)? The other thing is that at present sysbuild is focused around zephyr and zephyr-projects only, in the future we might want support for external projects/build systems that are not zephyr-based, and would want a want of supporting the custom target names for those systems

Reply:
if we want to map any target of images, then a more centralized place could make sense.

But we should be very careful not to start a whack-a-mole game, so I think this comment / proposal belongs in the discussion here: https://github.com/zephyrproject-rtos/zephyr/issues/53607#issuecomment-1378528024.